app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,27 @@
 //@import "bootstrap-sprockets";
 @import "bootstrap";
 
-.text {
-  height: 200px;
+body {
```

Comment:
@linko, 
It's problem of 3-rd bootstrap related with fixed navbar and sticky footer.
http://stackoverflow.com/a/11124805/4102471 - fixed navbar
http://stackoverflow.com/questions/17966140/twitter-bootstrap-3-sticky-footer - Sticky footer.(It's not the my solution, but idea the same)